src/style.css

```
@@ -0,0 +1,20 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
```

если font, то тег html
```suggestion
html {
  font-family: Arial, Helvetica, sans-serif;
  font-size: 13px;
}
```